src/include/function/list/operations/list_extract_operation.h

```diff
@@ -26,10 +26,12 @@ struct ListExtract {
     template<typename T>
     static inline void operation(
         ku_list_t& list, int64_t pos, T& result, ValueVector& resultValueVector) {
-        auto uint64Pos = (uint64_t)pos;
-        if (list.size < uint64Pos) {
-            throw RuntimeException("list_extract(list, index): index=" + TypeUtils::toString(pos) +
-                                   " is out of range.");
+        if (pos == 0 || abs(pos) > list.size) {
```

Comment:
This seems to be incorrect. You are returning without setting any value for result which leaves it into an undefined state. The right thing should be setting resultValueVector.setNull(pos or pos - 1) to make sure result is a null value.

Add test so see if you can reproduce the undefined state case.